src/stylesheets/screens/lazy-loader.scss

```diff
@@ -0,0 +1,111 @@
+.skeleton {
+  $self: &;
+
+  &-header {
+    display: flex;
+    height: auto;
+    margin-top: rem(32px);
+    padding: rem(0 32px);
+    width: 100%;
+  }
+
+  &__menu-bar {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+  }
+
+  &__menu-bar .react-loading-skeleton {
+    border-radius: rem(12px);
+    height: rem(18px);
+    width: rem(117px);
+  }
+
+  &__user-img {
+    margin-left: auto;
+    order: 2;
+  }
+
+  &__user-img .react-loading-skeleton {
+    border-radius: 50%;
+    height: rem(36px);
+    width: rem(36px);
+  }
+
+  &-details {
+    display: flex;
+    flex-direction: column;
+    max-width: rem(704px);
+    padding: rem(50px 32px);
+    @include media-breakpoint(map-get($grid, 'desktop')) {
+      padding-top: rem(50px);
+    }
+    @include media-breakpoint(map-get($grid, 'tablet')) {
+      padding-top: rem(50px);
+    }
+  }
+
+  &__date .react-loading-skeleton {
+    border-radius: rem(12px);
+    height: rem(18px);
+    width: rem(117px);
+  }
+
+  &__day {
+    margin-top: 14px;
+  }
+
+  &__day .react-loading-skeleton {
+    border-radius: 12px;
+    height: rem(18px);
+    width: rem(90px);
+  }
+
+  &__survey-img {
+    margin-top: rem(65px);
+  }
+
+  &__survey-img .react-loading-skeleton {
+    border-radius: rem(12px);
+    height: rem(305px);
+  }
+
+  &__survey-title {
+    margin-top: rem(38px);
+    max-width: rem(318px);
+  }
+
+  &__survey-title .react-loading-skeleton {
+    border-radius: rem(12px);
+    height: rem(18px);
+    width: rem(318px);
+  }
+
+  &__survey-desc {
+    margin-top: rem(8px);
+  }
+
+  &__survey-desc .react-loading-skeleton {
+    border-radius: rem(12px);
+    height: rem(18px);
+    width: rem(207px);
+  }
+
+  &__navigation {
+    margin-left: auto;
+    order: 2;
+    position: relative;
+    @include media-breakpoint(map-get($grid, 'desktop')) {
```

Comment:
the value for `desktop` and `mobile` are the same?

Reply:
For `desktop` and `tablet`, I put the same value of `bottom`. For tried the mobile-first approach as said in the compass.

Reply:
@namtx Should I keep it as it is?

Reply:
If we don't have different values for each screen, we should omit the `media-breakpoint` I think.

Reply:
The thing is, for mobile screens I'm using no `bottom` attribute. I'm using it for desktop and tablet which has the same `bottom` value. What should be done in these cases? @namtx

Reply:
so just `@include media-breakpoint(map-get($grid, 'tablet')) {}` is enough